frontend/packages/erd-core/src/features/erd/components/ERDContent/components/TableNode/TableDetail/Indexes/IndexesItem/IndexesItem.tsx

```diff
@@ -7,16 +7,25 @@ type Props = {
     name: string
     unique: boolean
     columns: string[]
+    type: string
   }
 }
 
 export const IndexesItem: FC<Props> = ({ index }) => {
+  const HIDE_INDEX_TYPE = 'btree'
+
   return (
     <div className={styles.wrapper}>
       <dl className={styles.dl}>
         <div className={styles.dlItem}>
           <dt className={clsx(styles.dt, styles.dtOnly)}>{index.name}</dt>
         </div>
+        {index.type && index.type.toLowerCase() !== HIDE_INDEX_TYPE && (
+          <div className={styles.dlItem}>
+            <dt className={styles.dt}>Type</dt>
+            <dd className={styles.dd}>{index.type}</dd>
```

Comment:
The display differs slightly depending on the parser, but there is no exact correct answer, so the original schema notation is respected.